pkg/network/protocols/http2/dynamic_table.go

```diff
@@ -21,8 +21,6 @@ import (
 
 const (
 	terminatedConnectionsEventStream = "terminated_http2"
-
-	defaultMapCleanerBatchSize = 1024
```

Comment:
I'd simply change it to 1

Reply:
It was currently serving only the HTTP/2 protocol. I moved it to the protocols file and used the same constant for all USM protocols.